Builds/VisualStudio2019/README.md

```diff
@@ -6,6 +6,12 @@ We do not recommend Windows for rippled production use at this time. Currently,
 the Ubuntu platform has received the highest level of quality assurance,
 testing, and support. Additionally, 32-bit Windows versions are not supported.
 
+While there are several options for building `rippled` on Windows, we recommend using
+[Chocolatey, Conan, CMake and Visual Studio](README.md#build-using-chocolatey-conan-cmake-and-visual-studio-recommended).
+If using this option, you can skip the section [Install Software](README.md#install-software)
+because Conan automatically installs all dependencies for `rippled`.
+
+
```

Comment:
Editing this document reflects a totally different philosophy on how to present build instructions in this repo. I've got a waiting [PR](https://github.com/XRPLF/rippled/pull/4381) to remove this file.

Reply:
Is it better to revert all changes for `Builds/VisualStudio2019/README.md` and add Windows-specific instructions to `BUILD.md`?